weblink.rst

```diff
@@ -0,0 +1,282 @@
+WebLink
+=======
+
+Symfony natively supports `Web Linking`_. It is especially useful to improve
+the performance of your application by leveraging the HTTP/2 protocol and
+preloading capabilities of modern web browsers.
+
+By implementing cutting edge web standards, namely `HTTP/2 Server Push`_ and
+W3C's `Resource Hints`_, the WebLink component
+brings great opportunities to boost webapp's performance.
+
+Thanks to WebLink, HTTP/2 (**h2**) servers are able to push resources to clients
+before they even know that they need them (think to CSS or JavaScript
+files, or relations of an API resource). WebLink also enables other very
+efficient optimisations that work with HTTP 1:
+
+-  telling the browser to fetch or to render another webpage in the
+   background ;
+-  init early DNS lookups, TCP handshakes or TLS negotiations
+
+Let's discover how easy it is to use it and the real life benefits you
+can expect.
+
+To benefit from HTTP/2 Server Pushes, a HTTP/2 server and a HTTPS connection
+are mandatory (even in local).
+Both Apache, Nginx and Caddy support these protocols.
+Be sure they are properly configured before reading.
+
+Alternatively, you can use the `Docker installer and runtime for
+Symfony`_ provided by Kévin Dunglas (community supported).
+
+It includes everything you need to run Symfony
```

Comment:
not sure about this part + the below instruction: looks redundant with the readme of symfony-docker

Reply:
removed